src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/ParsersCommon.cs

```diff
@@ -435,11 +435,11 @@ void SkipDigits(bool signAllowed)
             }
             else
             {
-                string subString = _pathString.Substring(start, _curIndex - start);
+                ReadOnlySpan<char> slice = _pathString.AsSpan(start, _curIndex - start);
 
                 try
                 {
-                    return System.Convert.ToDouble(subString, _formatProvider);
+                    return double.Parse(slice, provider: _formatProvider);
```

Comment:
Not directly related to your change, but use `double.TryParse` and throw in the false-branch?
Thus avoiding the use of exceptions (has runtime cost) for control-flow?

Reply:
I agree. It would be better. I've thought about this, but didn't do this because I want my PR to be as simple as possible.

Reply:
👍 makes sense. We can do this is a follow-up PR.